### DISCUSSION TITLE
Partially blind users

### DISCUSSION THREAD

**Discussion by claudiux** [linuxmint/discussions#105] [Questions]
My 85-year-old father is partially blind. He suffers from ARMD (age-related macular degeneration) and cannot tolerate bright light. I try to set Cinnamon for him, but nothing satisfies him fully. 
What theme, colors, type of icons etc. do you recommend?
Thank you for him and all partially blind users!

**Comment by mtwebster** [comment]
Can he tell you (or has a doctor discussed) more tolerable colors?  I can't find anything online so I guess there's no 'magic' combination.

What about adjusting the brightness (like on a laptop display) or altering the display gamma - you can do quite a bit with redshift settings.

**Comment by claudiux** [comment]
Thank you @mtwebster I'll make more tests.
